.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
+OPENAI_API_KEY=
```

Comment:
Instead of hardcoding the tokens, consider prompting users to enter them if they are not already set, providing a more flexible way to manage environment settings.